src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -425,6 +426,9 @@ void primitive_inst::set_arguments() {
 }
 
 void primitive_inst::build_deps() {
+    if (_node == nullptr)
+        return;
```

Comment:
Shouldn't exception be thrown here?

Reply:
https://github.com/openvinotoolkit/openvino/pull/13986/files#r1024718320
Updated to throw an exception when _node is null. Thank you.